practice-two/src/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style/main.css">
+  <title>Document</title>
+</head>
+<body>
+  <!--START HEADER-->
+  <div class="container">
+    <header class="main-header">
+      <nav class="navbar">
+        <p class="nav__logo">H.</p>
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a href="javascript:void(0)" class="item-content ">Prices</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="javascript:void(0)" class="item-content ">Booking</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="javascript:void(0)" class="item-content">Blog</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="javascript:void(0)" class="item-content">Services</a>
+          </li>
+          <div class="nav__list-icon">
+            <img src="assets/images/search-icon.png" alt="icon search" class="icon-search" />
```

Comment:
You should use a div to wrap image. You can use svg for icon instead of png image.